app/models/database.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+from boto3.session import Session
+from app.service.url_shortener import Link
+import os
+
+#this file will contain connection to the database, and database logic
+
+
+LINKS_TABLE_NAME = "LinkShortener"
```

Comment:
Rather than have these as standalone functions using a global variable, I recommend the following. 

1. Create a class `class LinkDatabase` 
2. Give it the following `__init__(self, endpoint_url:str, table_name: str)`. Here you'll initialize the `session,dynamodb, table` instances.
3. Make your other functions instance methods `def create_link(self, link: Link) -> None` 

Then to call this you can create an instance `link_db = LinkDatabase` and use that throughout your code base.